patsy/desc.py

```diff
@@ -209,7 +234,7 @@ def _pretty_repr_(self, p, cycle): # pragma: no cover
                                 [self.intercept, self.intercept_origin,
                                  self.intercept_removed, self.terms])
 
-    __getstate__ = no_pickling
+    # __getstate__ = no_pickling
```

Comment:
Why does `IntermediateExpr` need to support pickling? I think it should only exist as a temporary object in memory?